src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -3658,7 +3674,7 @@ private static void checkValidForByteConversion(final SmallestDataStorageKibibyt
         {
             throw new ConfigurationException(String.format("%s must be positive value < %dB, but was %dB",
                                                            name,
-                                                           value.getUnit()
+                                                           DataStorageSpec.DataStorageUnit.BYTES
```

Comment:
can't you just do `Integer.MAX_VALUE`?

Reply:
Yeah, I was correcting it and testing and thinking between KB and B message and at the end forgot to clean. Sorry about that